examples/prototype_usage.py

```diff
@@ -0,0 +1,97 @@
+# Note: an alternative name for the package could be "hermes" (Hermes in Greek mythology was the god of searches)
+from fasttextsearch import TextSource, Transcript, SourcedText, infix_align
+
+#
+# Input data for the script.
+#
+# This is possibly a several hours long audio recording accompanied by
+# a text file that contains the full text content of the recording.
+# The text is expected to be:
+# - encoded in UTF-8 (i.e. contains non-ASCII symbols),
+# - not normalized (i.e., contains punctuation, including non-ASCII punctuation/whitespaces)
+# - not a 100% accurate transcription (doesn't account for repetitions/restarts/other disfluencies)
+# - not in a specific format (e.g. a line could be a whole chapter or a single sentence)
+audio_path = "bible.wav"
+text_path = "bible.txt"
+
+#
+# The script's output is a single JSONL file with segmentation in a format like the following:
+# $ cat bible.jsonl
+# {'id': random_id, 'audio': "bible.wav", 'begin_time'="00:00:00:000", 'end_time': "0:00:01:220",'text': "bible.txt", 'begin_byte': 0, 'end_byte': 7}
+# {'id': random_id, 'audio': "bible.wav", 'begin_time'="00:00:01:220", 'end_time': "0:00:01:760",'text': "bible.txt", 'begin_byte': 7, 'end_byte': 11}
+# ...
+out_path = "bible.jsonl"
+
+#
+# The ASR model to recognize the text and timestamps
+#
+asr = Asr(...)
```

Comment:
I'm proposing that we make this a separate process, that dumps transcripts to disk, for easier debugging and tuning.